mqtt/policy/src/core/mod.rs

```diff
@@ -348,6 +348,15 @@ impl From<EffectOrd> for Decision {
     }
 }
 
+impl From<&Statement> for EffectOrd {
```

Comment:
> Used to do value-to-value conversions while **_consuming_** the input value.
The client of the `Statement` should decide where to clone data before consumption. https://doc.rust-lang.org/std/convert/trait.From.html